modules/promotion/src/Plugin/Commerce/PromotionOffer/PromotionOfferInterface.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\commerce_promotion\Plugin\Commerce\PromotionOffer;
+
+use Drupal\commerce_order\AdjustableInterface;
+use Drupal\commerce_price\Price;
+use Drupal\Component\Plugin\ConfigurablePluginInterface;
+use Drupal\Component\Plugin\PluginInspectionInterface;
+use Drupal\Core\Executable\ExecutableInterface;
+use Drupal\Core\Plugin\ContextAwarePluginInterface;
+use Drupal\Core\Plugin\PluginFormInterface;
+
+/**
+ * Defines an interface for Offer plugins.
+ */
+interface PromotionOfferInterface extends ExecutableInterface, PluginFormInterface, ConfigurablePluginInterface, PluginInspectionInterface, ContextAwarePluginInterface {
```

Comment:
This is stupid, because that has to do with conditions and is a different PR to be made.